settings.js

```diff
@@ -175,6 +181,12 @@ function generate(){
         },
       },
       "filter" : {
```

Comment:
I wish this `AMPERSANDPLACEHOLDER` replacement wasn't necessary but understand why it is.